### PR TITLE
Updating upgrading editions.md

### PR DESCRIPTION
**Pull request by mdbiscan**
Updated the built-in components to use angle bracket syntax. See https://github.com/ember-learn/guides-source/issues/692

### CODE REVIEW

File: guides/release/upgrading/editions.md [hunk 1]
```diff
@@ -1854,13 +1854,13 @@ bindings will not work:
   This does not work, since @value is
   an argument and is immutable
 -->
-{{input value=@value}}
+<Input value=@value />
```
Comment (mdbiscan) [inline guides/release/upgrading/editions.md]: This doesn't seem right...

Comment (locks) [reply guides/release/upgrading/editions.md]: Should be `<Input @value={{@value}} />`

Comment (locks) [inline guides/release/upgrading/editions.md]:
```suggestion
<Input @value={{@value}} />
```

File: guides/release/upgrading/editions.md [hunk 2]
```diff
@@ -1854,13 +1854,13 @@ bindings will not work:
...
 
 <!--
   Instead, we should update the value
   by passing an _action_ to the component
 -->
-{{input value=@value key-up=@updateValue}}
+<Input value=@value key-up=@updateValue />
```
Comment (locks) [inline guides/release/upgrading/editions.md]:
```suggestion
<Input @value={{@value}} @key-up={{@updateValue}} />
```

**Comment by mdbiscan** [comment]
> To make straight conversions, you need to prefix arguments with @, and then you need curlies on the right hand side, so something like <Input @value={{this.componentProperty}}.

I get that. I was taken aback on `@value={{@value}}`. Thanks for the protips!

**Comment by locks** [comment]
Thanks for the help!

**Pull request merged**